lib/postgrestore.js

```diff
@@ -21,11 +22,17 @@ function PostgreStore(conString, options) {
     if(!conString){
         throw new Error('Connection String is missing.');
     }
-	this._options = options || {};
-	this._options.pgstore = this._options.pgstore || {};
+
+    this._options = options || {};
+    this._options.pgstore = this._options.pgstore || {};
+    this._difficulty = this._options.pgstore.difficulty || 10;
 	this._table = this._options.pgstore.table || 'passwordless';
     pg.defaults.poolSize = this._options.pgstore.pgPoolSize || 10;
 
+    if(!isNumber(this._difficulty) || this._cost < 1) {
+        throw new Error('bcrypt difficulty must be an integer >= 1');
```

Comment:
Technically difficulty=0 should be allowed too (which would do one round of bcrypt), but the way the `bcrypt` node library works makes that just use the default difficulty.